dnsdos/dnsdos.rb

```diff
@@ -0,0 +1,108 @@
+##
+# This module requires Metasploit: http//metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
```

Comment:
Can you add comments describing the attack? Or link to an article explaining it.